### ISSUE TITLE
[feature request] Azure Dev Ops

### ISSUE THREAD

**Issue by gparlakov** [continuousauth/web#5]
Hey, @MarshallOfSound I like the lib you build and I think it solves the safe OTP proxy problem! 

I would like to add Azure Dev Ops (AzDO)provider (requester). 

It looks like to that end several things need to happen:
 - implement an AzDORequester that would persist the AzDO data
 - add some UI in the /client part to enable users to add AzDO projects
 - (optional) Update the node-client (adjacent repo) to allow for AzDO (as it now seems to hard-code support TravisCI and CircleCI only)

What do you think?

**Comment by MarshallOfSound** [comment]
Yup those are the three things that need to be done for new requesters.  Not super familiar with AZDO but I'll review any PRs you send my way :smile:
